apps/remix-ide/src/app/panels/layout.ts

```diff
@@ -92,4 +102,16 @@ export class Layout extends Plugin {
     this.panels[name].minimized = minimized
     this.event.emit('change', null)
   }
+
+  async maximiseSidePanel () {
+    this.event.emit('maximisesidepanel')
+    const current = await this.call('sidePanel', 'currentFocus')
```

Comment:
unclear what this does? where is the implementation?

Reply:
it is defined here: https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/src/app/components/panel.ts#L17